release-team/role-handbooks/docs/README.md

```diff
@@ -746,7 +746,8 @@ Announce that `[future release]` branch is open for new feature docs on slack #s
 
 - Clean Up
 ⚠️  Assign any issues / feature / PRs (that missed the release) to the new Docs Lead
-⚠️  Create a PR to remove shadows from [milestone maintainers](https://github.com/orgs/kubernetes/teams/kubernetes-milestone-maintainers) 
+⚠️  Create a PR to remove lead and shadows from [milestone maintainers](https://github.com/orgs/kubernetes/teams/website-milestone-maintainers/)
```

Comment:
Leads stay in the list until the K8s release website version they led is supported officially. Do you mind clarifying that?